src/main/scala/io/applicative/datastore/util/reflection/Field.scala

```diff
@@ -0,0 +1,3 @@
+package io.applicative.datastore.util.reflection
+
+case class Field[T >: Any](name: String, value: T)
```

Comment:
[T] is treated as [T >: AnyRef] while Long, Byte, Int and other wrappers of primitive types are successors of AnyVal.